DapperDoodle/Exceptions/InvalidDatabaseTypeException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace DapperDoodle.Exceptions
+{
+    public class InvalidDatabaseTypeException : ArgumentException
+    {
+        public InvalidDatabaseTypeException() : base("The database you have selected is not valid, or is not supported yet.")
+        {
+            
```

Comment:
Remove blank line for consistency

Reply:
@Krummelz Thank you for the feedback. 👍🏼